src/lib/mina_base/test/mina_base_test.ml

```diff
@@ -0,0 +1,115 @@
+open Alcotest
+open Mina_base
+
+let () =
+  run "Test mina_base."
+    [ Zkapp_account_test.
+        ( "zkapp-accounts"
+        , [ test_case "Events pop after push is idempotent." `Quick
+              (checked_pop_reverses_push (module Zkapp_account.Events))
+          ; test_case "Actions pop after push is idempotent." `Quick
+              (checked_pop_reverses_push (module Zkapp_account.Actions))
+          ] )
```

Comment:
What do you think of having a cleaner "main" file? Here's my proposal:

in `zkapp_account_test.ml`, define 
```
let tests = [ test_case "Events pop after push is idempotent." `Quick
              (checked_pop_reverses_push (module Zkapp_account.Events))
          ; test_case "Actions pop after push is idempotent." `Quick
              (checked_pop_reverses_push (module Zkapp_account.Actions))
          ]
```
so that the main file only has 
```
("zkapp-accounts", Zkapp_account_test.tests)
```
The same principle would apply to the other test files.

Reply:
I considered it, but decided it's actually nice to have an extensive list of tests all in one place. A reader looking for a particular test, unsure where it would be, can now search through that single file to find the right test module, while after extracting tests lists to their respective modules, they would have to `grep` the entire directory to do the same thing.

Reply:
This is even more handy if the reader is unsure whaether the test exists in the first place and is not sure how it would be named, hence there is no searching phrase to use. One needs to read test descriptions one by one.